src/_guides/language/effective-dart/style.md

```diff
@@ -1,15 +1,14 @@
 ---
-layout: guide
 title: "Effective Dart: Style"
-description: "Formatting and naming rules for consistent, readable code."
-
+description: Formatting and naming rules for consistent, readable code.
 nextpage:
   url: /guides/language/effective-dart/documentation
-  title: "Documentation"
+  title: Documentation
 prevpage:
   url: /guides/language/effective-dart
-  title: "Overview"
+  title: Overview
 ---
+<?code-excerpt  replace="/ellipsis;/.../g;/\/\/!//g;/\/\*(\s*\.\.\.\s*)\*\//$1/g"?>
```

Comment:
My regex fu is lacking. What does this do?

Reply:
There are three (semicolon separated) replace expressions:

- `/ellipsis;/.../g`, replaces `ellipsis;` with `...`.
- `/\/\/!//g`, erases `//!`, which is the marker I use to force a line break when `dartfmt` doesn't want to add one :).
- `/\/\*(\s*\.\.\.\s*)\*\//$1/g`: essentially replaces `/* ... */` with `...`.